index.md

```diff
@@ -1,57 +1,198 @@
+							<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> MacOS</li>
+							<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> Linux</li>
+						</ul>
+					</div>
+					<div class="col-12 col-md-4">
+						<h5>Mobile</h5>
+						<ul class="list-unstyled">
+							<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> Android</li>
+							<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> iOS</li>
+							<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> iPadOS</li>
+						</ul>
+					</div>	
+					<div class="col-12 col-md-4">
+						<h5>Console <i class="color-monogame fs-4">*</i</h5>			
+						<ul class="list-unstyled">
+							<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> Playstation 4</li>
+							<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> Playstation 5</li>
+							<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> Xbox One</li>
+							<li class="mb-3 lead text-nowrap"><i class="bi bi-check-lg color-monogame pe-2"></i> Nintendo Switch</li>
+						</ul>
+					</div>								
+				</div>
+			</p>
+		</div>
+	</div>
+	<div class="row">
+	 <p class="text-secondary">
+	 	<i class="color-monogame fs-4">*</i> Console exporting requires access to the SDK for that console.  After completing the sign up processes for the console, your representative will be able to walk you through the steps of gaining accesses to the tools for exporting MonoGame to that console.
+	</p>
+	</div>
+</section>
+
+<section class="container-xxl my-5">
+	<div class="row">
+		<div class="col-12 col-md-6">
+			<h3 class="fw-semibold mb-4 text-body-emphasis">Open-Source</h3>
+			<p class="lead">
+				All the code is available to you ensuring you'll have the ability to make changes when you need to or even port to whole new platforms. This means MonoGame is free to use, forever.
+		   </p>
+		   <ul class="list-unstyled">
+			<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> No licensing cost!</li>
+			<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> No royalties!</li>
+			<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> No runtime fee!</li>
+		   </ul>
+		</div>
+		<div class="col-12 col-md-6 d-flex flex-row justify-content-center">
+			<img id="github-logo" class="featurette-image object-fit-contain" loading="lazy">
+		</div>		
+	</div>
+</section>
+
+
+<section class="container-xxl my-4">
+	<h2>Get Involved</h2>
+	<p class="lead">
+		With 1000s of shipped games we have built up a vibrant community of developers which use MonoGame for both fun and profit.
+	</p>
+    <div class="row align-items-md-center g-5 py-5">
+      <div class="col">
+        <div class="row g-4">
+          <div class="col-12 col-md-4 d-flex flex-column gap-2 mb-5">

```

Comment:
It may be nice if the images in these sections centered instead of left:

![image](https://github.com/MonoGame/monogame.github.io/assets/103522/03f8a988-f1da-4942-b184-a4b8d544c224)

Especially in the mobile layouts.

Reply:
These are now center aligned using `align-self: center`.  If the parent flex container used `align-content: center` it would center the icons and all text. https://github.com/MonoGame/monogame.github.io/pull/19/commits/c98c5706073753289af01ba67df7c7aecf694c6b